test/cpp/interop/server_helper.cc

```diff
@@ -41,7 +41,8 @@ std::shared_ptr<ServerCredentials> CreateInteropServerCredentials() {
   } else if (FLAGS_use_alts) {
     return GetCredentialsProvider()->GetServerCredentials(kAltsCredentialsType);
   } else if (FLAGS_use_tls) {
-    return GetCredentialsProvider()->GetServerCredentials(kTlsCredentialsType);
+    return GetCredentialsProvider()->GetServerCredentials(
+        kTls13CredentialsType);
```

Comment:
Why is this only TLS 1.3? Is this the thing you were referring to that made the Cloud-to-Cloud tests fail?

(If not, could you point me to the thing that made them fail?)

Reply:
We had to choose to default to something. I changed it back to default to minVersion=1.2 and maxVersion=1.3